src/settings/PatronNotices/components/EditSections/PatronNoticeAboutSection/PatronNoticeAboutSection.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import { Field } from 'react-final-form';
+import { memoize } from 'lodash';
+
+import {
+  Checkbox,
+  Col,
+  Row,
+  Select,
+  TextArea,
+  TextField,
+} from '@folio/stripes/components';
+
+import { patronNoticeCategories } from '../../../../../constants';
+import { validateUniqueNameById } from '../../../../utils/utils';
+
+const PatronNoticeAboutSection = ({ initialValues, okapi, intl }) => {
+  const { formatMessage } = intl;
+  const categoryOptions = patronNoticeCategories.map(({ label, id }) => ({
+    label: formatMessage({ id: label }),
+    value: id,
+  }));
+  const isActive = initialValues && initialValues.active;
+
+  const getTemplatesByName = (name) => {
+    return fetch(`${okapi.url}/templates?query=(name=="${name}")`,
```

Comment:
cannot mutator be used here instead of fetch?

Reply:
Here went lot of modularizations. This piece was an existing code.